www/js/components/device-storage.js

```diff
@@ -1,43 +1,49 @@
-import React, {Component} from 'react';
+/*global cordova*/
```

Comment:
The linter has a no-unused-variable rule. You have to tell the linter you are using globals.